docs/octopus-cloud/index.md

```diff
@@ -21,27 +21,24 @@ You can sign up for Octopus Cloud at [octopus.com/register](https://octopus.com/
 An Octopus account lets you manage your instances of Octopus Cloud.
 
 1. Enter your name.
-1. Provide your email address and click **Create a password**. Please note, these credentials are for you Octopus Account. You will also [create credentials](#create-a-cloud-instance) for your Octopus Cloud instance, when you create it.
-1. On the next screen, provide your company name.
-1. Chose a secure password and enter it twice.
-1. Click **Create my Octopus account**.
+1. Provide your email address and create a password. Please note, these credentials are for your Octopus Account and you will use them to log into your Octopus instance through single sign-on.
+1. On the next screen, verify your email address.
+1. After your email has been verified, you will be logged into your Octopus Account.
 
 ## Create a Cloud Instance
 
-1. From the instances screen, click **Create cloud instance**.
+1. From the **products** screen, click **Start a free 30-day Cloud trial**.
 1. Enter an **instance name** for your Octopus Cloud instance.
 1. Choose a URL for the instance.
 1. Select the Cloud region for your instance. Currently the only option is **US - Oregon**.
 1. Click **Enter account details**.
 1. Create your first user for Octopus Cloud.
-1. Enter the username the user will use to log into Octopus Cloud.
-1. Create a password for the user and confirm the password.
 1. Click **Continue to Confirmation**.
 1. Confirm the details you've provided, agree to the terms and click **Looks good. Deploy my Octopus!**.
 
-You will be taken to the account provisioning screen. Please note it can take five to ten minutes for your Octopus Cloud instance to be ready. You will receive an email when the instance is ready to use.
+You will be taken to the account provisioning screen. It should take a couple of minutes for your Octopus Cloud instance to be ready. You will receive an email when the instance is ready to use.
```

Comment:
```suggestion
You will be taken to the account provisioning screen. It will take a couple of minutes for your Octopus Cloud instance to be ready. You will receive an email when the instance is ready to use.
```